curriculum/test/test-challenges.js

```diff
@@ -132,8 +132,17 @@ setup()
   .catch(err => handleRejection(err));
 
 async function setup() {
-  if (process.env.npm_config_superblock && process.env.npm_config_block) {
-    throw new Error(`Please do not use both a block and superblock as input.`);
+  if (
+    (process.env.npm_config_superblock && process.env.npm_config_block) ||
+    (process.env.npm_config_superblock && process.env.npm_config_filename) ||
+    (process.env.npm_config_block && process.env.npm_config_filename) ||
+    (process.env.npm_config_block &&
+      process.env.npm_config_filename &&
```

Comment:
Here, you are looking at `filename`, but below you are checking for `dashedname`. I presume you initially wanted to use the filename, but then noticed that info wasn't part of the challenge info.

I would use the challenge `id`. Many of the new RWD challenges have the same name - e.g. `step-1`, `step-2` - so `dashedname` wouldn't really work for those. The `id` should work for any type of challenge.

Reply:
That is indeed what happened 😄  Sorry, was out of town and didn't have a computer, but the PR has been updated and should address this, thanks for the feedback!